Source/public/js/script.js

```diff
@@ -0,0 +1,13 @@
+function changeLocale(locale) {
```

Comment:
Kijk hier even naar:
https://stackoverflow.com/a/13062316

Reply:
Ik gebruik geen document.ready en hij staat niet in de <head> dus wat moet er hiermee gedaan worden?

Reply:
@JoeyBlankendaal Integreren met jouw code, om mogelijke errors te voorkomen.